tools/ci_build/github/azure-pipelines/c-api-noopenmp-packaging-pipelines.yml

```diff
@@ -164,38 +138,6 @@ stages:
     - template: templates/component-governance-component-detection-steps.yml
       parameters :
         condition : 'succeeded'
-
-
-- stage: Download_Java_Tools
```

Comment:
This part hase been moved to 
tools/ci_build/github/azure-pipelines/stages/jobs/steps/download-java-tools-step.yml